styles/banner.css

```
@@ -1,2 +1,199 @@
-.banner {
+/*setting*/
+.slider {
+  width: 90%;
+  height: 450px;
+  /* border: 2px solid black; */
+  margin: auto;
+  margin-top: 60px;
+  margin-bottom: 60px;
+  width: 100%;
+  font-family: "Pretendard-Regular", Arial, Helvetica, sans-serif; /*폰트 추가*/
+}
+.slide {
+  float: left;
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+
+/* second banner */
+
+.rect {
```

다른 요소와 충돌을 방지하기 위해서 앞에 `.slider` 붙여주실게요 :)
```suggestion
.sliders .rect {
```